src/Command/ServiceCommandlist.php

```diff
@@ -17,6 +17,6 @@ public function addCommandReference($command): void
 
     public function getCommandList(): array
     {
-        return $this->commandList;
+        return array_filter($this->commandList);
```

Comment:
Do you know why it became necessary to add this? Would it be possible to prevent empty items from being added to the list, rather than remove them every time the accessor is called?

Reply:
@greg-1-anderson Spent some time investigating it and did not found a real cause of this. What I found is that the dependency references for `config.export.commands` and `config.export.commands` turining into `NULL` once added which causes fatal error on Drupal10 CI pipeline but not on Drupal9-related ones.